### PR TITLE
Add doc for resolveSync()

### PR DESCRIPTION
**Pull request by unional**
Feel free to add anything to clarify. 🌷

### CODE REVIEW

File: docs/system-api.md [hunk 1]
```diff
@@ -138,6 +138,11 @@ SystemJS registry object supporting:
 
 See also [SystemJS.newModule](#systemjsnewmoduleobject---module).
 
+#### SystemJS.resolveSync(moduleName)
```
Comment (guybedford) [inline docs/system-api.md]: Optional second argument here too - `moduleName, [parentName])`

Perhaps something like synchronous alternative to `SystemJS.resolve`.

Comment (unional) [inline docs/system-api.md]: Sure. Speaking of which, `SystemJS.resolve()` is also missing from the doc.

Comment (unional) [inline docs/system-api.md]: Added

**Comment by guybedford** [comment]
Sure, thanks!

**Pull request merged**